### PR TITLE
OAuth2 frontend client removed, now uses cookie in frontend

### PR DESCRIPTION
**Pull request by Portals**
Also upgrades to rdc 1.0

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -42,9 +43,12 @@ services:
             POSTGRES_PASSWORD: password
             POSTGRES_USER: imageit
             POSTGRES_DB: imageit
-    gamma_backend:
-        build:
-            context: ./backend
-            dockerfile: dev.Dockerfile
+
+    redis:
+        image: bitnami/redis:5.0.8
```
Comment (Gurr1) [inline docker-compose.yml:48]: This is not the official Redis image. Any reason for this?

File: docker-compose.yml [hunk 2]
```diff
@@ -42,9 +43,12 @@ services:
...
+        restart: always
+        environment:
+            REDIS_PASSWORD: password
+        network_mode: host
```
Comment (Gurr1) [inline docker-compose.yml:52]: Any reason for running this on the host network?

Comment (Portals) [reply docker-compose.yml:52]: Since the backend is running on the host, right?

File: docker-compose.yml (hunk 1 above)
Comment (Portals) [reply docker-compose.yml:48]: Just because of the ENV feature that bitnami/redis gives

File: docker-compose.yml (hunk 2 above)
Comment (Gurr1) [reply docker-compose.yml:52]: no, forwarding the port is enough in this case

**Pull request merged**